README.md

```diff
@@ -1 +1,15 @@
-Smith Tech Page - Link to Kanban Git
+Smith Tech Page 
+* We use bootstrap for this website, before starting you may have to run the following commands in the scs-app/ folder: 
+
+1) npm install popper.js --save 
+2) npm install query --save
+3) npm install tether --save 
+4) npm install bootstrap --save 
+5) npm install bootstrap-icons --save
+
```

Comment:
Suggestion: add 
6) npm install jquery --save

Reply:
ah good one - I forgot to add that will re-raise pr or put in separate commit

Reply:
realized the query was a typ-o lol

Reply:
Ohh my gosh i should've realized that lol sorry

Reply:
haha np was my typo anyway lol